### ISSUE TITLE
Hitboxes are offset on ultrawide

### ISSUE THREAD

**Issue by SwarnaD** [WistfulHopes/SF6Mods#8]
![image](https://github.com/WistfulHopes/SF6Mods/assets/24879435/07aabd73-b182-466d-baa8-4fb2f1c53fb3)
![image](https://github.com/WistfulHopes/SF6Mods/assets/24879435/f360e6de-d15f-4334-8ecb-27bd6dec4042)

Hitboxes display funkily on ultrawide monitors in borderless mode. This is borderless on 5120*1440. They move faster to the edges than the characters themselves do. Looks like the calculations are being done based on the window edges rather than the actual render boundaries?

**Comment by WistfulHopes** [comment]
yeah, the calculations are done in screen space. i'll see if i can figure out world space transforms
